index.bs

```diff
@@ -5531,6 +6182,26 @@ partial dictionary MLOpSupportLimits {
         - an {{MLOperand}}. The output tensor of the same shape as *input*.
 </div>
 
+<table id=constraints-prelu class='data' link-for="MLGraphBuilder/prelu(input, slope, options)">
+  <caption>Constraints for {{MLGraphBuilder/prelu()}}</caption>
+  <thead>
+    <tr>
+      <th>input operand</th>
+      <th>[=/allowed data types=]</th>
+      <th>[=/allowed ranks=]</th>
+    </tr>
+  </thead>
+  <tr>
+    <td>{{input}}</td>
+    <td>{{MLOperandDataType/"float32"}}, {{MLOperandDataType/"float16"}}, {{MLOperandDataType/"int32"}}, {{MLOperandDataType/"int8"}}</td>
+    <td>[=/any rank|N=]</td>
+  <tr>
+    <td>{{slope}}</td>
+    <td>[=/same as=] {{input}}</td>
+    <td>0 to {{input}}'s [=MLOperand/rank=]</td>
```

Comment:
Sorry, could you please restore the change to "any rank"? As we discussed in https://github.com/webmachinelearning/webnn/pull/657#discussion_r1837478497

Reply:
done in 8e540f3b047ec16f49eee284c4686d59bd917925 !